### DISCUSSION TITLE
linguistic error in FashionMNIST DataSet

### DISCUSSION THREAD

**Discussion by yanivboker** [mrdbourke/pytorch-deep-learning#614] [General]
I believe there is a linguistic error in FashionMNIST DataSet.
It has a label called "Trouser" but it should be "Trousers".
Trousers is the singular and the plural form of pants.

**Comment by mrdbourke** [comment]
Hi @yanivboker , 

Good catch! You'll often find small errors like this in public datasets.

However, since I didn't create the dataset, I'm unable to change the original.

That being said, it would be possible in your own code to remap the naming of `Trouser` to `Trousers`.

**Reply by shisirkha** [reply]
Yeah i saw also many errors in FashionMNIST so i shifted to MNIST

**Reply by shisirkha** [reply]
Now my model can predict my pretty bad hand written numbers👍.
